sirepo/sim_data/ml.py

```diff
@@ -62,3 +63,57 @@ def _lib_file_basenames(cls, data):
         if name:
             return [cls.lib_file_name_with_model_field("dataFile", "file", name)]
         return []
+
+
+class ArchiveManager:
+    import contextlib
+
+    _ARCHIVE_EXTENSIONS = (
+        ".tar.gz",
+        ".zip",
+    )
+
+    def __init__(self, file_path):
+        import zipfile
```

Comment:
These can be global.

Reply:
[3c866a1](https://github.com/radiasoft/sirepo/pull/4736/commits/3c866a1fe074693237b4be774648761a1e03b649)